polaris.shopify.com/pages/examples/page-full-width.tsx

```diff
@@ -1,5 +1,5 @@
-import {Page, LegacyCard} from '@shopify/polaris';
-import {PlusIcon} from '@shopify/polaris-icons';
+import {Page, Card, Text} from '@shopify/polaris';
```

Comment:
```suggestion
import {Page, Card, Text, BlockStack} from '@shopify/polaris';
```

To resolve possible CI errors with suggestion to fix Card spacing above.